### DISCUSSION TITLE
Option to disable HTTPS in WebRTC

### DISCUSSION THREAD

**Discussion by KopiNinja** [bluenviron/mediamtx#1312] [Questions] [answered]
Hello, your project is amazing!
I can share my camera and sometimes my pc on other local devices in a very easy way, thank you very much!
I would like to ask if you could add an option to disable HTTPS in WebRTC.
The reason is that whenever I need to share on TV or with devices of people I know, there is always a warning and for not knowing about, they are scared or afraid of something bad like a virus.
So I would like to disable HTTPS to use on the local network.
Projects like [rtmp-to-webrtc](https://github.com/Sean-Der/rtmp-to-webrtc), [go2rtc](https://github.com/AlexxIT/go2rtc), [RTSPtoWebRTC](https://github.com/deepch/RTSPtoWebRTC) and [RTSPtoWeb](https://github.com/deepch/RTSPtoWeb) work fine without HTTPS.
If it's not too much work, I'd like to ask for that option.
Thanks and sorry for the mistakes, it's not my native language.

**Comment by aler9** [comment, marked as answer]
done. https://github.com/aler9/rtsp-simple-server/commit/ec864010374a9fac78933454ac2f6d341f00e534

**Reply by KopiNinja** [reply]
Wow! That was fast, you're a really nice guy. Now my parents and uncles won't complain about the warning haha
Thank you so much!

[a comment by an automated account is not shown]
